financial-templates-lib/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -187,6 +189,14 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
     this.historicalPricePeriods = newHistoricalPricePeriods;
     this.lastUpdateTime = currentTime;
   }
+
+  _invertPriceSafely(priceBN) {
+    if (priceBN && priceBN.gt(this.toBN("0"))) {
+      return this.toBN(this.toWei("1"))
+        .mul(this.toBN(this.toWei("1")))
+        .div(priceBN);
+    }
```

Comment:
Two nits:
1. Is it intended that this returns `undefined` in the case that the `if` is false? If so, it might be a little clearer to return it explicitly below the `if`, so it doesn't look like a forgotten return.
2. This is a pretty tiny nit because we don't really deal with negative prices, but there's nothing wrong with inverting a negative number, right? It might be a little clearer to check for `!priceBN.isZero()` rather than `priceBN.gt(this.toBN("0"))` as well as a bit simpler to read.

Reply:
>Is it intended that this returns undefined in the case that the if is false? If so, it might be a little clearer to return it explicitly below the if, so it doesn't look like a forgotten return.

Yes that's the intent. I'll make that explicit

>This is a pretty tiny nit because we don't really deal with negative prices, but there's nothing wrong with inverting a negative number, right? It might be a little clearer to check for !priceBN.isZero() rather than priceBN.gt(this.toBN("0")) as well as a bit simpler to read.

True! WIll change